packages/manager/modules/hub/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@ovh-ux/manager-hub-components",
```

Comment:
We used to have the following link between apps and modules:

`@ovh-ux/manager-<PACKAGE_NAME>-app` (standalone application) that rely on `@ovh-ux/manager-<PACKAGE_NAME>` (module business logic).

My point is what not using `@ovh-ux/manager-hub` as module name?

Reply:
It was a way to indicate that it was a set of components but not a full module (with routing and layout as those will be handled by app)